### PR TITLE
docs: correct ngSwitch definition

### LINKED ISSUE #35484: ngSwitch is an attribute directive or structural directive
# 📚 Docs or angular.io bug report

### Description
Surfing the documentation and found 2 description for ngSwitch
in https://angular.io/guide/template-syntax#the-ngswitch-directives : 
i found it attribute directive
in the others i found it as structural directive 

is thers something i didnt understand well ?

### What's the affected URL?**
https://angular.io/guide/template-syntax#the-ngswitch-directives

## 📷Screenshot
Annotation 2020-02-17 041234
Annotation 2020-02-17 041234

### PR DESCRIPTION
**Pull request by santoshyadavdev**
Fixes #35484

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #35484

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/common/src/directives/ng_switch.ts [hunk 1]
```diff
@@ -96,7 +94,7 @@ export class SwitchView {
  * ```
  *
  * @publicApi
- * @see `NgSwitchCase`
+ * @see `NgSwitch`
```
Comment (sacgrover) [inline packages/common/src/directives/ng_switch.ts]: It should be NgSwitchCase as we are already reading NgSwitch API doc

**Comment by sacgrover** [comment]
I think, In NgSwitch just replace "structural directive" to "attribute directive"

File: packages/common/src/directives/ng_switch.ts (hunk 1 above)
Comment (santoshyadav198613) [reply packages/common/src/directives/ng_switch.ts]: Actually the docs are swapped, this one is for ngSwitch and https://github.com/angular/angular/blob/master/packages/common/src/directives/ng_switch.ts#L186 this one is for ngSwitchCase.

**Comment by santoshyadav198613** [comment]
> I think, In NgSwitch just replace "structural directive" to "attribute directive"

ngSwitch is an attribute directive, ngSwitchCase is structural directive.

**Comment by santoshyadav198613** [comment]
Hi @kapunahelewong ,
Can you review please.

**Comment by santoshyadav198613** [comment]
Hi @kapunahelewong ,
Can we get it reviewed, let me know if any changes are required.

**Comment by santoshyadav198613** [comment]
Hi Team, 
Can you review this one?

File: packages/common/src/directives/ng_switch.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/common/src/directives/ng_switch.ts:97]: These "see" changes do need to be reverted. See the preview

Comment (santoshyadav198613) [reply packages/common/src/directives/ng_switch.ts]: done.

File: packages/common/src/directives/ng_switch.ts [hunk 2]
```diff
@@ -159,7 +157,9 @@ export class NgSwitch {
 /**
  * @ngModule CommonModule
  *
- * @description
+ * @description A structural directive that adds or removes templates (displaying or hiding views)
+ * when the next match expression matches the switch expression.
+ *
```
Comment (petebacondarwin) [inline packages/common/src/directives/ng_switch.ts:162]: To be honest I don't really see much benefit in this sentence. The following text describes the `NgSwitchCase` directive adequately IMO.

**Review by petebacondarwin — APPROVED** [review]
Approved but I would suggest just removing the extra description.

File: packages/common/src/directives/ng_switch.ts (hunk 2 above)
Comment (santoshyadav198613) [reply packages/common/src/directives/ng_switch.ts:162]: The problem was it describes ngSwitch as Structural directive, so I changed that, do you want me to remove this text?

Comment (santoshyadav198613) [reply packages/common/src/directives/ng_switch.ts:162]: ok got it. thanks

Comment (santoshyadav198613) [reply packages/common/src/directives/ng_switch.ts]: Done.

**Comment by petebacondarwin** [comment]
I think @kapunahelewong could global-docs-approve this one and then we could remove @kara  from the required list of approvers.

**Review by kapunahelewong — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Comment by matsko** [comment]
presubmit = https://test.corp.google.com/ui#id=OCL:308083436:BASE:308083469:1587663581693:203913ee

(I know this isn't necessary, but we're just using this PR as a test to see if things are passing)

**Comment by AndrewKushnir** [comment]
FYI, g3 presubmit was successful (only unrelated failures).

**Pull request closed without merging**